locale/en/invitation.po

```diff
@@ -101,4 +101,274 @@ msgid "invitation.userRoleAssignment.userGroup.startDate.mustBeAfterToday"
 msgstr "This attribute must have a value equal or after today"
 
 msgid "invitation.validation.error.propertyProhibited"
-msgstr "The :attribute field is prohibited"
\ No newline at end of file
+msgstr "The :attribute field is prohibited"
+
+msgid "invitation.step"
+msgstr "STEP"
+
+msgid "invitation.header"
+msgstr "Invitations"
+
+msgid "invitation.inviteToRole.btn"
+msgstr "Invite to a role"
+
+msgid "invitation.wizard.pageTitle"
+msgstr "Invite user to take a role"
+
+msgid "userInvitation.enterDetailsLabel"
+msgstr "Enter details"
+
+msgid "userInvitation.reviewAndInviteLabel"
+msgstr "Review & invite for roles"
+
+msgid "userInvitation.searchUser.stepName"
+msgstr "Search User"
+
+msgid "userInvitation.searchUser.stepLabel"
+msgstr "{$step} - Search User"
+
+msgid "userInvitation.searchUser.nextButtonLabel"
+msgstr "Search User"
+
+msgid "userInvitation.searchUser.stepDescription"
+msgstr "Search for the user using their email address, username or ORCID ID. Enter at least one details to get started. If the user does not exist, you can invite them to take up roles and be a part of your journal. If the user already exist in the system, you can view user information and invite to take a additional roles."
+
+msgid "userInvitation.emailField.description"
+msgstr "e.g. aeinstein@example.com"
+
+msgid "userInvitation.usernameField.description"
+msgstr "e.g. mickeymouse"
+
+msgid "userInvitation.orcidField.description"
+msgstr "e.g. 0000-0000-0000-0000"
+
+msgid "userInvitation.enterDetails.stepName"
+msgstr "Enter details"
+
+msgid "userInvitation.enterDetails.stepLabel"
+msgstr "{$step} - Enter details and invite for roles"
+
+msgid "userInvitation.enterDetails.nextButtonLabel"
+msgstr "Save And Continue"
+
+msgid "invitation.role.selectRole"
+msgstr "Select a new role"
+
+msgid "invitation.role.dateStart"
+msgstr "Start Date"
+
+msgid "invitation.role.dateEnd"
+msgstr "End Date"
+
+msgid "invitation.role.masthead"
+msgstr "Journal Masthead"
```

Comment:
Anything journal-related should probably move to the applications as well. If you've already done this for the OJS language, could you do this for the word "journal" as well, @withanage?